lib/databases/timescale/lootboxes/rewards.go

```diff
@@ -20,16 +22,74 @@ type UserLootboxCount struct {
 	LootboxCount float64 `json:"lootbox_count"`
 }
 
+// buildUserRewardValuesString to build a variadic string for the values to insert for top winners
+func buildUserRewardValuesString(userCount int) (string, error) {
+	if userCount < 1 || userCount > 10 {
+		return "", fmt.Errorf(
+			"invalid number of leaderboard users - want 1-10, got %d",
+			userCount,
+		)
+	}
+
+	uniqueMessages := []string{
+		`($2, '', 'leaderboard_prize', $1, 0, 1, 30, 'none'),
```

Comment:
Think you could store this instead of a slice of strings, instead with a struct then spit out using the templating language